package.json

```diff
@@ -47,31 +49,30 @@
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
-    "prettier": "^1.3.1",
-    "react": "^15.4.2",
-    "react-dom": "^15.5.0",
-    "react-test-renderer": "^15.5.0",
+    "prettier": "^1.7.3",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-test-renderer": "^16.0.0",
     "semantic-release": "^6.3.6"
   },
   "peerDependencies": {
     "patternfly": "^3.27.4",
     "prop-types": "^15.5.10",
-    "react": "^15.5.0"
+    "react": "^15.5.0 || ^16.0.0",
```

Comment:
should we bump to 15.6.2? it will at least show deprecation warnings but will work?

Reply:
I am OK with either (as long as we can support downstreams). I will be trying to update all of my projects to 16 soon. Any thoughts on this version @jtomasek ?

Reply:
@ohadlevy I see why you are saying 15.6.2 now...from the release notes:
https://facebook.github.io/react/blog/2017/09/26/react-v16.0.html

> We've been serving React 16 to Facebook and Messenger.com users since earlier this year, and we released several beta and release candidate versions to flush out additional issues. With minor exceptions, if your app runs in 15.6 without any warnings, it should work in 16.

Reply:
updated to 15.6.2 @ohadlevy

Reply:
I would not be so strict in terms of bumping peerDependencies as it forces the consuming apps to constantly update too. It definitely makes sense in case of React 16, although I'd keep 15.5.0 for now. TripleO UI is currently updating to  15.6.2 to get under MIT license. We won't be able to update for some time as we're currently hitting a few deprecation warnings on dependencies (Mostly Formsy-react which we're moving from towards redux-form).